app/assets/javascripts/app/phone-internationalization.js

```diff
@@ -17,7 +17,13 @@ const areaCodeFromUSPhone = (phone) => {
   return null;
 };
 
-const unsupportedPhoneOTPDeliveryWarningMessage = (phone) => {
+const selectedInternationCodeOption = () => {
+  const dropdown = document.querySelector('#two_factor_setup_form_international_code');
+  const option = dropdown.item(dropdown.selectedIndex);
```

Comment:
small nit: seems silly to define this variable only to return it on the next line?

could be:

```
return dropdown.item(dropdown.selectedIndex);
```